tests/upgrades/test_virtwho.py

```diff
@@ -171,19 +166,17 @@ def test_post_crud_virt_who_configuration(self):
             2. the config and guest connection have the same status.
             3. virt-who config should update and delete successfully.
         """
-        org = entities.Organization().search(query={'search': f'name={self.org_name}'})[0]
+        org = entities.Organization().search(query={'search': f"name={org_data['name']}"})[0]
 
         # Post upgrade, Verify virt-who exists and has same status.
         vhd = entities.VirtWhoConfig(organization_id=org.id).search(
-            query={'search': f'name={self.name}'}
+            query={'search': f"name={form_data['name']}"}
```

Comment:
Please switch these to single-quote strings.

Reply:
Hey, Jake, a little confused, why we have to switch single-quote strings?

Reply:
Done, thanks

Reply:
Hey, Jake, a little confused, why we have to switch it to single-quote strings?

Reply:
Hey there Kunxin, single quotes are the standard defined in robottelo's code standards doc.
https://github.com/SatelliteQE/robottelo/blob/master/docs/code_standards.rst

Reply:
Got it, Thank you